rs/registry/canister/canister/canister.rs

```diff
@@ -205,16 +205,17 @@ fn get_changes_since() {
         }
 
         // Parse request.
-        let request = deserialize_get_changes_since_request(arg_data())
+        let (version, limit) = deserialize_get_changes_since_request(arg_data())
             .map_err(|err| (Code::MalformedMessage, err.to_string()))?;
-        let version = request;
+        let limit = limit.unwrap_or(MAX_VERSIONS_PER_QUERY as u32) as usize;
```

Comment:
Would it be possible to add a limit in bytes instead? Unless it's too difficult?

Reply:
@sasa-tomic makes sense :) Not sure however this is how NNS team wants it to be implemented. Will wait for their feedback